homeassistant/components/nws/weather.py

```diff
@@ -107,7 +107,6 @@ def __init__(self, entry_data, hass_data, mode, units):
             self.coordinator_forecast = hass_data[COORDINATOR_FORECAST_HOURLY]
         self.station = self.nws.station
 
-        self.is_metric = units.is_metric
```

Comment:
I couldn't find any use for this variable in nws, so I have removed it completely.

Reply:
Looks to be leftover after unit conversions were centralized.